Jenkinsfile

```diff
@@ -20,7 +20,12 @@ pipeline {
     choice(name: 'engineListUrl',
       description: 'Engine to use for build',
       choices: ['http://zugprojenkins/job/ivy-core_product/job/master/lastSuccessfulBuild/',
-                'http://zugprobldmas/job/Trunk_All/lastSuccessfulBuild/'])
+                'http://zugprobldmas/job/Trunk_All/lastSuccessfulBuild/',
+                'http://developer.axonivy.com/download/maven.html'])
+
+    string(name: 'engineVersion',
+      description: "Version of the engine under test: must match 'enginListUrl' provided version",
+      defaultValue: '8.0.1' )
 

```

Comment:
with every new release we have to increase this version here?

Reply:
yeah not sexy. I ship a better approach...